src/Features/Core/Portable/SymbolSearch/SymbolSearchOptions.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Runtime.Serialization;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.SymbolSearch;
+
+[DataContract]
+internal readonly record struct SymbolSearchOptions()
+{
+    [DataMember]
+    public bool SearchReferenceAssemblies { get; init; } = true;
+
+    [DataMember]
+    public bool SearchNuGetPackages { get; init; } = true;
+
+    public static readonly SymbolSearchOptions Default = new();
+}
+
+internal static class SymbolSearchOptionsStorage
+{
+    private static readonly OptionGroup s_optionGroup = new(name: "symbol_search", description: FeaturesResources.Symbol_search);
```

Comment:
maybe dotnet_symbol_search?  what's the namign we're generally using for these groups?

Reply:
"dotnet" is part of the option name. It ends up as `dotnet.symbolSearch.searchReferenceAssemblies` in VS Code after the transformations we do.